README.md

```diff
@@ -77,6 +78,7 @@ Once the testing environment is fully operational, the following services will b
 - Hadoop YARN: http://hadoop-master1.orb.local:8088
 - Hadoop MapReduce JobHistory: http://hadoop-master1.orb.local:19888
 - Ranger Admin: http://hadoop-master1.orb.local:6080 (admin/Ranger@admin123)
+- Trino Web UI: http://hadoop-master1.orb.local:18081 (admin/)
```

Comment:
I remember there is no username/password by default
```suggestion
- Trino Web UI: http://hadoop-master1.orb.local:18081
```

Reply:
Currently, it needs a user but no password, if there is no additional configuration.

Reply:
The description is here: https://trino.io/docs/current/admin/web-interface.html#authentication